### ISSUE TITLE
30 day removal notice - China source

### ISSUE THREAD

**Issue by papjul** [breezy-weather/breezy-weather#510]
As pointed out by an user in #500, China source is actually a Xiaomi service.
Except a few unofficial documentations, it doesn't seem to be open to be used by anyone (a few users documented it on Internet, allowing Geometric Weather to use it initially).
However, this means we are not allowed to use it, and in case there is any breaking change in this service, we won't be able to fix it.

As a conclusion, this source will be removed in 30 days.

As an alternative, we suggest using any other source. If hitting China firewall, other alternatives are currently studied:
- QWeather. Work in progress in #207

You're free to suggest other alternatives, excluding the following:
- ColorfulClouds / Caiyun. No free tier: https://caiyunapp.com/api/weather_intro.html
- Seniverse. No free tier: https://www.seniverse.com/pricing

**Comment by GrimPixel** [comment]
Both ColorfulClouds and Seniverse have a free tier, “免费版” in Chinese. The free tier requires application, just like other services.

**Comment by papjul** [comment]
I don't speak Chinese but I was told you only get 10 000 calls for lifetime in the free-tier. Not 10 000 calls per day. Once you are over with the 10 000 calls (which you usually reach in a day or two), you can't use it anymore without paying, so it's not free, it's a 10 000 calls trial.

For Seniverse, please reply in #508

**Comment by papjul** [comment]
This is a really bad idea for security reasons. This source is mostly based on ColorfulClouds / Caiyun data, which is indeed known as one of the best for China, but QWeather data should also be good, this is what Apple Weather uses, for reference.

**Comment by itsHenry35** [comment]
> This is a really bad idea for security reasons. This source is mostly based on ColorfulClouds / Caiyun data, which is indeed known as one of the best for China, but QWeather data should also be good, this is what Apple Weather uses, for reference.

Yeah can understand you but literally apple's weather is not accurate at all in china at least in my region, i once used ios and the weather annoyed me a lot

**Comment by papjul** [comment]
Dark Sky API was replaced by Apple Weather Kit recently, which is based on QWeather data now, so maybe try again, or have a look on QWeather directly website if it feels accurate

**Comment by itsHenry35** [comment]
> Dark Sky API was replaced by Apple Weather Kit recently, which is based on QWeather data now, so maybe try again, or have a look on QWeather directly website if it feels accurate

Fair enough, though not as accurate as the current source, at least better than accuweather. Gotta ask if you guys are actually working on it?

**Comment by papjul** [comment]
See #207 if interested

**Comment by papjul** [comment]
Due to China country requirements when signing up for API keys, we have no alternative at the moment. The source will be removed if we have, although I have no government-approved alternative sources at the moment
